### PR TITLE
Al 1254 add previous ami to socify event

### PR DESCRIPTION
**Pull request by rruster**
- Send previous AMI ID when invoking Socify Event
- Move the AMI ID to the data section of the Socify Json Object

### CODE REVIEW

**Comment by rruster** [comment]
build

**Comment by rruster** [comment]
build

File: disco_aws_automation/disco_deploy.py [hunk 1]
```diff
@@ -695,7 +704,8 @@ def update(self, dry_run=False, deployment_strategy=None, ticket_id=None):
                 raise
 
         socify_helper.send_event(status=status,
-                                 hostclass=(DiscoBake.ami_hostclass(ami) if ami else None),
+                                 hostclass=(hostclass if ami else None),
```
Comment (aoskotsky-amplify) [inline disco_aws_automation/disco_deploy.py]: If we got down here then `ami` has to be defined. We don't need this `if`

**Comment by rruster** [comment]
build

File: disco_aws_automation/disco_deploy.py (hunk 1 above)
Comment (rruster) [inline disco_aws_automation/disco_deploy.py]: When no ami is defined and a ticket was specify the code also send an event to Socify to add a comment to indicate that the deploy failed. So the test is needed.
If we don't need to update the Ticket in that case we can remove the call to Socify.

**Pull request merged**